### ISSUE TITLE
Polaris: galil jaws support

### ISSUE THREAD

**Issue by Tom-Willemsen** [ISISComputingGroup/IBEX#4377]
Various support and diagnostics relating to 3 galil controllers on polaris (jaw sets 1-5):

Before galil controller power cycle:
- IBEX was talking fine with the controller (verified by sending messages in engineering view)
- IBEX was unable to move any of the stages controlling jaw blades
- IBEX *was* talking to the controller OK (e.g. energise light came on and saw motor pulses being sent when a move was requested)
- IBEX *was* able to move a small test stage that electronics section brought along
- A jog box plugged directly into the galil was unable to move the jaws

After a galil controller power cycle (on all 3 controllers, followed by IOC restarts and re-homes) the motion returned to normal. I am not sure why this helped or what state the galils had managed to get into.
